types/lolex/lolex-tests.ts

```diff
@@ -16,25 +16,14 @@ lolex.createClock<lolex.NodeClock>(new Date());
 lolex.createClock<lolex.NodeClock>(7, 9001);
 lolex.createClock<lolex.NodeClock>(new Date(), 9001);
 
-lolex.install<lolex.BrowserClock>(7);
-lolex.install<lolex.BrowserClock>(new Date());
-lolex.install<lolex.BrowserClock>(7, ["setTimeout"]);
-lolex.install<lolex.BrowserClock>(new Date(), ["setTimeout"]);
-
-lolex.install<lolex.NodeClock>(7);
-lolex.install<lolex.NodeClock>(new Date());
-lolex.install<lolex.NodeClock>(7, ["setTimeout"]);
-lolex.install<lolex.NodeClock>(new Date(), ["setTimeout"]);
-
-lolex.install<lolex.BrowserClock>({}, 7);
-lolex.install<lolex.BrowserClock>({}, new Date());
-lolex.install<lolex.BrowserClock>({}, 7, ["setTimeout"]);
-lolex.install<lolex.BrowserClock>({}, new Date(), ["setTimeout"]);
-
-lolex.install<lolex.NodeClock>({}, 7);
-lolex.install<lolex.NodeClock>({}, new Date());
-lolex.install<lolex.NodeClock>({}, 7, ["setTimeout"]);
-lolex.install<lolex.NodeClock>({}, new Date(), ["setTimeout"]);
```

Comment:
Why remove so many tests?

Reply:
I removed the tests because the corresponding signatures don't exist anymore